src/store/model/msi.ts

```diff
@@ -0,0 +1,253 @@
+import {Store} from './store';
+
+export const MSI: Store = {
+  currency: '$',
+  labels: {
+    inStock: {
+      container: '#button-cart',
```

Comment:
And for those of you camping out for the pre-order launch ([since MSI isnt shipping on launch day](https://videocardz.com/newz/msi-delays-geforce-rtx-5090-official-us-store-availability-to-february-6))

```js
inStock: {
    // Unknown what the pre-order text looks like so lets see what sticks!
    container:
      '#product_qty button:not([onclick^="compare.add"]):not([onclick^="notify.showNotify"])',
    text: [''],
  }
 ```
 should capture anything that isnt explicitly the existing compare or notify me buttons - this is too aggro for the production release though IMO